tests/rust/ptrs_as_arrays.rs

```diff
@@ -0,0 +1,11 @@
+/// cbindgen:ptrs-as-arrays=[[arg;3]]
```

Comment:
does it work with the normal `//` comment as well?

Reply:
Unfortunately no, because comments aren't parsed.

Reply:
I see, I tried it on rav1e and it seems working well for the purpose :)